devsetup/Makefile

```diff
@@ -175,10 +175,12 @@ edpm_deploy_instance: ## Spin a instance on edpm node
 edpm_play_cleanup: ## Cleanup EDPM openstackansibleee resource
 	-oc delete openstackansibleee deploy-external-dataplane-compute
 
-.PHONY: cifmw_prepare
-cifmw_prepare: ## Clone the ci-framework repository in the ci-framework directory. That location is ignored from git.
+./ci-framework:
 	git clone https://github.com/openstack-k8s-operators/ci-framework ci-framework
 
+.PHONY: cifmw_prepare
+cifmw_prepare: ./ci-framework ## Clone the ci-framework repository in the ci-framework directory. That location is ignored from git.
+
```

Comment:
yes here we can always just do 'cd ci-framewrok; git pull' (as this target depends on existance of ./ci-framework itself now)

Reply:
I wouldn't expect a dev to change things there in, and re-run `make cifmw_prepare` to be honest. Safeguards are always good though.

Reply:
It does conditional pull now - so should be safe even for modified env.